force-app/main/default/triggers/ThrowExceptionIfOppStageIsInvalid.trigger

```diff
@@ -0,0 +1,13 @@
+trigger ThrowExceptionIfOppStageIsInvalid on Opportunity (before update) {
+	for(Opportunity oppNew : Trigger.New) {
+        if(oppNew.StageName == 'Closed Won'){
+            for(Opportunity oppOld : Trigger.Old) {
```

Comment:
check ChangeOnQuoteSentStage trigger to avoid for in for loop and using old records